### PR TITLE
Add site impacting change entry for MutationEvents deprecation.

### PR DESCRIPTION
**Pull request by sanketj**
AB#44942023

### CODE REVIEW

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ This table lists:
...
 | Removal of cross-origin subframe JavaScript dialogs | Future release (TBD) | | Removes `window.alert`, `window.prompt`, and `window.confirm` from cross-origin iframes.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see [Intent to Remove: Cross origin subframe JS Dialogs](https://groups.google.com/a/chromium.org/g/blink-dev/c/hTOXiBj3D6A/m/JtkdpDd1BAAJ). |
+| Removal of mutation events | v127 | | Removes support for all mutation events in Chromium. For more details, see [Intent to Deprecate: Mutation Events](https://groups.google.com/a/chromium.org/g/blink-dev/c/qDsKRU-cQ_4/m/isA1mZ_aAAAJ). Mutation events have been deprecated per spec for over a decade and authors are recommended to use the `MutationObserver` API instead. |
```
[the remark this answers is not in the document]
Comment (sanketj) [reply microsoft-edge/web-platform/site-impacting-changes.md]: Thanks. I've included your suggestion.

File: microsoft-edge/web-platform/site-impacting-changes.md [hunk 2]
```diff
@@ -52,6 +52,7 @@ This table lists:
...
 | Ignore modifications to `document.domain` by default | v115 | | The `document.domain` property historically could be set to relax the same-origin policy and allow subdomains from a site to interact. This behavior will be disabled by default such that setting the `document.domain` property will have no effect.  For more information and workarounds, see [Microsoft Edge will disable modifying document.domain](/deployedge/edge-learnmore-origin-keyed-agent-cluster). |
 | Removal of cross-origin subframe JavaScript dialogs | Future release (TBD) | | Removes `window.alert`, `window.prompt`, and `window.confirm` from cross-origin iframes.  This change is happening in the Chromium project, on which Microsoft Edge is based.  For more information, see [Intent to Remove: Cross origin subframe JS Dialogs](https://groups.google.com/a/chromium.org/g/blink-dev/c/hTOXiBj3D6A/m/JtkdpDd1BAAJ). |
+| Removal of mutation events | v127 | | Removes support for mutation events in Chromium. Use the `MutationObserver` API instead.  See [Intent to Deprecate: Mutation Events](https://groups.google.com/a/chromium.org/g/blink-dev/c/qDsKRU-cQ_4/m/isA1mZ_aAAAJ). |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/site-impacting-changes.md:55]:
```suggestion
| Removal of mutation events | v127 | | Removes support for mutation events in Chromium. Use the [MutationObserver](https://developer.mozilla.org/docs/Web/API/MutationObserver) API instead.  See [Intent to Deprecate: Mutation Events](https://groups.google.com/a/chromium.org/g/blink-dev/c/qDsKRU-cQ_4/m/isA1mZ_aAAAJ). |
```
https://groups.google.com/a/chromium.org/g/blink-dev/c/qDsKRU-cQ_4/m/isA1mZ_aAAAJ doesn't link to 
https://developer.mozilla.org/docs/Web/API/MutationObserver (as far as I can tell),
so linking here.

**Review by mikehoffms — APPROVED** [review]

**Comment by mikehoffms** [comment]
@sanketj - is this PR ready to merge/publish?

(having trouble contacting you in chat)

**Review by captainbrosset — APPROVED** [review]

**Comment by sanketj** [comment]
> @sanketj - is this PR ready to merge/publish?
> 
> (having trouble contacting you in chat)

Yes, it is. Thanks for the feedback.

**Pull request merged**